monai/transforms/compose.py

```diff
@@ -281,6 +285,9 @@ def __call__(self, data):
 
         - the channel dimension is not omitted even if number of channels is one
 
+        Raises:
+            NotImplementedError: Subclass {self.__class__.__name__} must implement the compute method
```

Comment:
most of your error messages have "." at the end, do you want to add "." for all of them?

Reply:
Yeah, I can update the error messages